test/unit/units/parser.cpp

```diff
@@ -125,6 +125,11 @@ SCENARIO("Unit parser accepting valid units definition", "[unit][parser]") {
                 REQUIRE(is_valid_construct("dipotre\t\t\t/m\n"));
             }
         }
+        WHEN("A 's' is added") {
+            THEN("parser remove it to find the units") {
+                REQUIRE_NOTHROW(parse_string("pew 1 m\nfoo 2 pews\n"));
```

Comment:
I'm curious about this. Is there such case in the `nrnunits.lib`?

Reply:
no, should I remove test and code related to that?